src/main/java/com/project/pufferfish/screen/InvadersScreen.java

```diff
@@ -34,6 +36,14 @@
     int relX = 0;
     int relY = 0;
 
+    
+    //Player variables
+    public int playerWidth=15, playerHeight=15;
+    Player p = new  Player (textureWidth/2-playerWidth/2,textureHeight-(2*playerHeight));
```

Comment:
Rename 'p' variable to something more defined, like "userSpaceShip" or "invaderPlayer". This way others can know what 'p' is in the class

Reply:
Of course, if you have a better variable name, it'd be good to have it here

Reply:
renamed p to tank